install.sh

```diff
@@ -62,7 +62,13 @@ function install_macos(){
     fi
     # Install the required packages
     echo "Installing required Packages" 
-    brew install python@3.10 tcl-tk python-tk
+    if [! commad --version python3 &> /dev/null ]; then
```

Comment:
I could be wrong, but should this not be command not commad?

If not dw, just looks like a spelling mistake to me